src/test/app/ValidatorSite_test.cpp

```diff
@@ -137,13 +125,10 @@ class http_sync_server
     void
     on_accept(error_code ec)
     {
+        // ec must be checked before `acceptor_` or the member variable may be
+        // accessed after the destructor has completed
         if(ec || !acceptor_.is_open())
```

Comment:
So previously `ec` was checked after `acceptor`, which of course is a bug.  Nice find.